site/gatsby-site/i18n/locales/es/validation.json

```diff
@@ -0,0 +1,10 @@
+{
+  "*URL required": "*URL requerida",
+  "*Title is required": "*Título requerido",
+  "*Author is required. Anonymous or the publication can be entered.": "*Autor requerido. Anónimo o la publicación puede ser introducido",
+  "*Date published is required": "*Fecha de publicación requerida",
+  "*Date downloaded required": "*Fecha de descarga requerida",
+  "*Must enter URL in http://www.example.com/images/preview.png format": "*Debe introducir la URL en el formato http://www.ejemplo.com/imágenes/preview.png",
```

Comment:
@cesarvarela What this `*` means in this context? Are they part of the markdown of the page? If they are, should we remove the markdown from the translations to keep the page styles in another place?

Reply:
No special meaning it is just the * of "hey this is required"